packages/flutter_tools/test/drive_test.dart

```diff
@@ -297,10 +302,18 @@ void main() {
         Device device = await findTargetDevice();
         expect(device.name, 'mock-android-device');
       }, overrides: <Type, Generator>{
-        FileSystem: () => memoryFileSystem,
+        FileSystem: () => fs,
       });
+    }
+
+    group('findTargetDevice on Linux', () {
+      findTargetDeviceOnLinuxOrWindows(isLinux: true);
+    });
```

Comment:
Should this test be skipped on Windows? Likewise, should the test below be skipped on Linux?